packages/react/src/components/MenuButton/_menu-button.scss

```diff
@@ -1,5 +1,6 @@
 @import '~carbon-components/scss/components/menu/menu';
 @import '../../globals/vars';
+@import './menu-button-shadow-blocker';
```

Comment:
If we change these to `@use` will this fix the sass error you received?

Reply:
Nope, I did try that too

Reply:
See [comment here](https://github.com/carbon-design-system/carbon-addons-iot-react/pull/3185#discussion_r785830751)